src/main/java/org/jabref/gui/preferences/exporter/ExportCustomizationTab.fxml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.TableColumn?>
+<?import javafx.scene.control.TableView?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<fx:root spacing="10.0" type="VBox"
+         xmlns="http://javafx.com/javafx" xmlns:fx="http://javafx.com/fxml"
+         fx:controller="org.jabref.gui.preferences.exporter.ExportCustomizationTab">
+
+    <Label styleClass="titleHeader" text="%Custom export formats"/>
+
+    <TableView fx:id="exporterTable">
+        <columns>
+            <TableColumn fx:id="nameColumn" text="%Export name"/>
+            <TableColumn fx:id="layoutColumn" text="%Main layout file"/>
+            <TableColumn fx:id="extensionColumn" text="%Extension"/>
+        </columns>
+        <columnResizePolicy>
+            <TableView fx:constant="CONSTRAINED_RESIZE_POLICY"/>
+        </columnResizePolicy>
+    </TableView>
+
+    <HBox alignment="BASELINE_RIGHT" spacing="10.0">
+        <Button onAction="#add" text="%Add" prefWidth="80.0"/>
```

Comment:
Just a general remark: I think you can also directly call/bind to the viewmodel in the fxml. I think, the syntax was `#controller.viewmodel.add`. Simple one-way bindings can be done in a similar way.

Reply:
I tested that out, tried some variants and to find some documentation about this. Although in theory this should work, it seems like I cannot expose the methods of the viewModel to the fxml loader. When loading java throws a
`javafx.fxml.LoadException: Error resolving onAction='#controller.viewModel.fillColumnList', either the event handler is not in the Namespace or there is an error in the script.`
(Maybe because we load the ViewModels in the initialize method?)
I think it's better to leave this now as I already put some time now in this and the middle step is still common practice in our codebase. I'll focus on debugging the localization tests.

Reply:
After spending a couple of hours on debugging we finally found out that the afterburner ViewLoader is setting its own controller when calling ViewLoader.view(this)  And that is called from the constructor in the  dialogs.